src/modules/Header/components/LinkItemUpper.js

```diff
@@ -40,7 +40,7 @@ const LinkItemUpper = styled(NavLink)`
   }
   
   @media ${MEDIA_QUERIES.MOBILE} {
-    font-size: 16px;
+    font-size: 14px;
```

Comment:
Tailwind provides font sizes in rem units. px units shouldn't be used. Pixels don't scale.